### PR TITLE
Removed openseesvis package import and dependency

### PR DESCRIPTION
**Pull request by shihabkhan1**
Now package is able to install and work..

### CODE REVIEW

**Review by justinngan92 — COMMENTED** [review]
Approve

**Review by justinngan92 — APPROVED** [review]

**Review by justinngan92 — APPROVED** [review]

**Review by justinngan92 — APPROVED** [review]

File: src/ospgrillage/postprocessing.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 from ospgrillage.load import ShapeFunction
 from ospgrillage.static import solve_zeta_eta
 
-import openseespyvis.Get_Rendering as opsplt
```
Comment (ccaprani) [inline src/ospgrillage/postprocessing.py:20]: Can't delete this variable; it is used throughout codebase. #74 redfines it with vfo which is now accepted. Pull again and check it works.

**Comment by abdelhaq-ourosama** [comment]
Hello @shihabkhan1 ,
How were you able to successfully install ospgrillage?
I am encountering a dependency issue with "openseespyvis" even though I have installed "vfo".
Thanks in advance

**Comment by ccaprani** [comment]
@justinngan92 Similar to the other PR, can you rebase onto #74 and see if we can merge please?

**Comment by ccaprani** [comment]
Closed based on DM from @justinngan92 that this is now incorporated into #86 .

**Pull request closed without merging**